spinnaker.md

```diff
@@ -0,0 +1,268 @@
+```
+
+**Verify Installation**
+
+```
+hal -v
+```
+
+Move `hal` into your `$PATH`
+
+## Create a Github repository
+
+Later in this exercise you will be creating a Spinnaker pipeline that runs when code is pushed to a Github repository called spin-kub-repo.
+ 
+Login to Github and fork the [lwander/spin-kub-demo](https://github.com/lwander/spin-kub-demo) repository to your Github account.  
+
+> For directions on how to fork a repository, see https://help.github.com/articles/fork-a-repo/
+
+## Create a link to your Github repository from Docker Hub
+
+In this exercise you'll be using Docker's automated build service to create Docker images when code is pushed to your Github repository.  The first step is to link your Github repository from Docker Hub.  
+
+1. Log into Docker Hub.
+2. Navigate to **Your Login Name > Settings > Linked Accounts & Services**.
+3. Click **Link Github**.
+    The system prompts you to choose between Public and Private and Limited Access. Choose the **Public and Private** connection type since its required if you want to use the Automated Builds.
+4. Press **Select** under Public and Private connection type.
+    The system will prompt you to enter your Github credentials. 
+
+![linked-account](https://github.com/jicowan/aws-microservices-deploy-options/blob/master/images/linked-acct.png)
+
+After you grant access to you code repository, the system returns you to Docker Hub and the link is complete. 
+
+## Create an automated build repository
+
+Automated build repositories build an image when changes are pushed to your source code repository.  To create an automated build repository follow these instructions: 
+
+1. Select **Create** > **Create Automated Build** (from the drop down, upper right) on [Docker Hub](https://hub.docker.com/).
+2. Select **Github.**
+    The system prompts you with a list of User/Organizations and code repositories.
+3. Select your username from the User/Organizations list.
+4. Select `spin-kub-demo` repository from the list.
+5. Customize the automated build by pressing the **Click here to customize** this behavior link.
+    1. Enter “Spin Kub Build Demo” for **Short Description**.
+    2. Set the **Docker Tag value** for **Push Type: Branch** to latest
+    3. Add a** Push Type: Tag** with the Docker Tag Name of 1.0
+6. Click **Create**.
+7. Click **Build Settings.**
+8. Click **Trigger** to trigger the build manually.  
+
+## Add docker registry provider
+
+Enable the `docker-registry` provider:
+
+```
+hal config provider docker-registry enable
+```
+
+Add Docker Hub as the registry provider: 
+
+```
+hal config provider docker-registry account add dockerhub \
+--address index.docker.io \
+--repositories <your_dockerhub_username>/spin-kub-demo \
+--username <your_dockerhub_username> \
+--password
+```
+
+## Add a Kubernetes account
+
+Enable the Kubernetes (legacy) provider, enter the following commands in a terminal: 
```

Comment:
Why legacy instead of manifest provider?

Reply:
I couldn't find a good example that used the manifest provider.  This was easier.